modules/mobile/spec/services/v2/appointments/provider_names_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Mobile::V2::Appointments::ProviderNames do
+  let(:user) { FactoryBot.build(:iam_user) }
+  let(:subject) { described_class.new(user) }
+  let(:single_practitioner_with_name) do
+    [
+      {
+        "identifier": [
+          {
+            "system": 'dfn-983',
+            "value": '520647669'
+          }
+        ],
+        "name": {
+          "family": 'KNIEFEL',
+          "given": [
+            'CAROLYN'
+          ]
+        },
+        "practiceName": 'Cheyenne WY VAMC'
```

Comment:
where is this practiceName coming from? I'm not seeing it on VAOS v2 docs. was it from a VCR?

Reply:
I just removed this since it adds nothing to the test.